parsl/dataflow/dflow.py

```diff
@@ -82,7 +82,8 @@ def __init__(self, config=Config()):
         logger.debug("Starting DataFlowKernel with config\n{}".format(config))
 
         if sys.version_info < (3, 6):
-            logger.warning("Support for python versions < 3.6 is deprecated and will be removed after parsl 0.10")
+            logger.error("Support for python versions < 3.6 stopped after parsl 1.0.0")
```

Comment:
Is this necessary? With setup.py specifying `python_requires=">=3.6.0",` you won't be able to install parsl at all.

Reply:
i'm fine with removing it entirely

Reply:
Let's do that.